fvm/src/kernel/default.rs

```diff
@@ -466,6 +466,37 @@ where
         }))
     }
 
+    fn verify_bls_aggregate(
+        &self,
+        aggregate_signature: &[u8; BLS_SIG_LEN],
+        pub_keys: &[[u8; BLS_PUB_LEN]],
+        digests: &[[u8; BLS_DIGEST_LEN]],
+    ) -> Result<bool> {
+        let sig = Signature::new_bls(aggregate_signature.to_vec());
+        let pub_keys: Vec<&[u8]> = pub_keys.iter().map(AsRef::as_ref).collect();
+        let digests: Vec<&[u8]> = digests.iter().map(AsRef::as_ref).collect();
+
+        let gas_required = {
+            let gas_two_pairings = self.call_manager.price_list().sig_cost[&SignatureType::BLS]
+                .flat
+                .as_milligas();
+            let gas_one_pairing = gas_two_pairings / 2;
+            let num_pairings = digests.len() as u64 + 1;
+            let gas_required = Gas::from_milligas(num_pairings * gas_one_pairing);
+            GasCharge::new("OnVerifySignature", gas_required, Zero::zero())
+        };
```

Comment:
Currently, this is just an estimate of the gas cost for aggregate signature verification after hashing has been removed. I tried using the gas calibration test to measure what this should actually be, but wasn't able to get it working.

Reply:
That's fine, we can set it to zero for now and measure later.

Reply:
Sounds good, thanks.